### DISCUSSION TITLE
Ability to inverse the original/dup folders

### DISCUSSION THREAD

**Discussion by kaplandani** [hsoft/dupeguru#1186] [Ideas]
Lets assume I have 2 folders
c:\dani\original
c:\dani\backup

and they share duplicate files. 
if the scan shows the backup folder first - I can only mark the original "files" and delete them. 
I wish there was a way to inverse the results - so the original folder will be shown un-selectable and the backup folder will allow me to mark and delete.

**Comment by ziadoz** [comment]
I agree this feature would be really helpful.

I have some folders that contain photos I've dumped from a camera, and other folders where I've copied and sorted those folders by location. But DupeGuru is picking up the dump folder ones as being originals, and the sorted ones as being duplicates.

So if I have this:

| | Filename | Folder |
| --- | --- | --- |
| | Image-1.jpg | Camera/Dumps |
| [x] | Image-1.jpg | LocationA |
| | Image-2.jpg | Camera/Dumps |
| [x] | Image-2.jpg | LocationB |

It would be nice to be able to flip it:

| | Filename | Folder |
| --- | --- | --- |
| [x] | Image-1.jpg | Camera/Dumps |
| | Image-1.jpg | LocationA |
| [x] | Image-2.jpg | Camera/Dumps |
| | Image-2.jpg | LocationB |

**Comment by rkazak** [comment]
Is that not supported with the use of the "reference" State?  Mark the directory you want to keep as a reference and then the other will be flagged as a duplicate?

**Reply by user18081972** [reply]
what if the duplicate files are all in the same directory though?

**Comment by bwprager123** [comment]
My impression is that you can not decide in advance what the reference directory will be. This means if there are a few hundred dupes, you can only change the preferred file to reference and mark the ones you prefer to remove one file at a time. What I wish I knew how to do was to set one directory as containing the presumed preferred files to keep prior to the scan. What this would allow us to do is to pre-select for results the software can't otherwise possibly know. I want my "newer" versions of files to remain, for example, because those are compilations I personally selected and want those directories to remain while deleting the older, larger directories that are already backed up. THis cannot be done as far as I know. I hope to hear that it can. Thank you.

**Comment by rkazak** [comment]
Hi bwprager123,

I have linked a screen recording of ( hopefully ) what I think you mean you
would like to do.   Let me know what you think.

Regards
Rohinton.
 Screen Recording 2024-03-19 at 4.04.34 PM.mov
<https://drive.google.com/file/d/1EpSstKJ1oqpFhLTDEqIpKysPiB2VGr98/view?usp=drive_web>

**Comment by listlfa** [comment]
Also second. I also have a lot of media. Setting a 'default' reference folder would be great.
i have a single main folder. and some scattered other folders, some with duplicates. i made this mess by being lazy. but it would be nice to have an easier way to clean up.

**Reply by jombelo** [reply]
you should be able to choose EVERY file in a group for deletion if you want. that you can't choose some file in a group of dublicates for deletion, makes that program just unuseful for real life situations, i made just an issue for that..
